### DISCUSSION TITLE
Glench still MIA

### DISCUSSION THREAD

**Discussion by burkybang** [Glench/ExtPay#250] [General]
Related to #243, which was closed

I still can't get a response from @Glench via email. I really need support. The last time he responded was September 15, 2024. I'm posting here again because it was the only way I was able to receive any response from him.

**Comment by bilalsattar24** [comment]
I've had the same experience. Haven't heard from him in quite a long time. Concerning

**Comment by atreiou** [comment]
Yeah is there anyone else at all involved in this? I'm sure I saw that there are a couple of others involved? I literally just have to work out why Firefox is blocking the cookies and not letting the second payment page open with stripe, and I'm done, I can launch. So frustrating!

**Reply by burkybang** [reply]
In my 2+ years working with ExtensionPay, I have not seen anyone else other than Glen.

**Reply by bilalsattar24** [reply]
I'd highly recommend you roll your own payment system. Extension pay is starting to show issues. 

For example sometimes if a user purchases, extension pay still thinks that the user hasn't paid yet even though Stripe shows the transaction. 

This is very concerning. 

I will be creating my own payment system this summer.

**Reply by Glench** [reply]
> For example sometimes if a user purchases, extension pay still thinks that the user hasn't paid yet even though Stripe shows the transaction.

Hey! No one's ever mentioned this before. Do you have any reproduction steps or can you email a list of users where this has happened?

**Reply by bilalsattar24** [reply]
Good to see you back.

No idea how to reproduce it. I've had 7 different users reach out to me with that exact issue. @Glench

**Reply by Glench** [reply]
Well please email me a list of users where this happened.
